aptos-move/aptos-gas-schedule/src/gas_schedule/move_stdlib.rs

```diff
@@ -41,5 +41,8 @@ crate::gas_schedule::macros::define_gas_parameters!(
         [bcs_serialized_size_failure: InternalGas, { RELEASE_V1_18.. => "bcs.serialized_size.failure" }, 3676],
 
         [mem_swap_base: InternalGas, { RELEASE_V1_24.. => "mem.swap.base" }, 1500],
+
+        [vector_move_range_base: InternalGas, { RELEASE_V1_24.. => "vector.move_range.base" }, 4000],
+        [vector_move_range_per_index_moved: InternalGasPerArg, { RELEASE_V1_24.. => "vector.move_range.per_index_moved" }, 10],
```

Comment:
Wow this is pretty cheap. I hope you've done some crude benchmarks at least?

Reply:
In #14862, more concretely these are the results: https://gist.github.com/igor-aptos/e8d4e21edcbc75dddcb9382d4e077665

but yeah, memcopy is **extremely** cheap